demo/src/store/StoreHelpers.js

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
what this comment belongs to? The file itself? then please use
`@fileoverview` jsdoc
https://google.github.io/styleguide/jsguide.html#jsdoc-top-file-level-comments

Reply:
Thanks, I added the fileoverview tag